src/anonymous-credentials/schema.ts

```diff
@@ -262,7 +264,7 @@ export class CredentialSchema extends Versioned {
           f = Encoder.positiveIntegerEncoder();
           break;
         case CredentialSchema.INT_TYPE:
-          f = Encoder.integerEncoder(value['minimum']);
+          f = Encoder.integerEncoder(value['minimum'] || 0);
```

Comment:
fair enough, but instead of letting it crash, why not use int min value? not sure if you are using int32/int64 etc in rust code, perhaps you can shed some light and we can pass a valid value to cover negative integers too

Reply:
similar case for the number type, are we using f32 or f64?

Reply:
Because in practice, `minimum` is expected to be negative. So a minimum of zero will still crash for a negative integer. The proof system needs all values to be positive integers, hence the conversion in case of negative or decimal. More info here https://github.com/docknetwork/crypto-wasm-ts/#bound-check-using-legogroth16

Reply:
@lovesh you made your point about it cant be 0, that was a mistake on my part. however, we could default to integer minimum limits so that minimum doesnt always have to be specified in the schema. see latest updates

Reply:
i have removed number/int minimums here as its better to do in exploration PR